blogs/2020/07/27/containers-edu.md

```diff
@@ -0,0 +1,217 @@
+
+![More detailed Python devcontainer.json](18-python-devcontainer.png)
+
+`devcontainer.json` is a great opportunity to create a customized coding environment to fit the specific needs of your students. You can specify a few variables to create a unique environment for each class or assignment.
+
+Variables we'd recommend starting off with in a `devcontainer.json` for education are:
+
+| Property | Type | Description |
+|----------|------|-------------|
+| `image` | string | The name of an image in a container registry (i.e. [DockerHub](https://hub.docker.com)) that VS Code should use to create the dev container. |
+| `dockerFile` | string | The location of a [Dockerfile](https://docs.docker.com/engine/reference/builder/) that defines the contents of the container. The path is relative to the `devcontainer.json` file. You can find a number of sample Dockerfiles for different runtimes in the [vscode-dev-containers repository](https://github.com/microsoft/vscode-dev-containers/tree/master/containers). |
+| `name` | string | A display name for the container. |
+| `extensions` | array | An array of extension IDs that specify the extensions that should be installed inside the container when it is created. Defaults to `[]`. |
+| `settings` | object | Adds default `settings.json` values into a container/machine specific settings file.  |
+
+Setting up a handful of variables in this file will save your class time in the long run. For example, you can use the `extensions` variable to ensure all your students automatically get the same set of extensions installed. Setting paths can also be challenging or lead to issues on students' computers, but by specifying path information via the `settings` variable, you'll save your class time and ensure their computers' path variables aren't harmed in the process.
+
+We have a reference table [here](https://code.visualstudio.com/docs/remote/containers#_devcontainerjson-reference) of all available properties you can include in this configuration file.
+
+#### Dockerfile
+
+Docker can build images automatically by reading the instructions from your [Dockerfile](https://docs.docker.com/engine/reference/builder/). This file will at the very least have a base image specified using `FROM`, and it can include command-line instructions as well.
+
+Here's what the Dockerfile looks like in our Python app:
+
+![Python dockerfile in app](19-dockerfile-cropped.png)
+
+### Opening the container
+
+Once our container configuration files were added, we got a notification that our folder now has a `devcontainer.json` file. Let's select **Reopen in Container**:
+
+![Reopen in container notification](21-reopen-notif.png)
+
+A new instance of VS Code launches, and we can see our image is being built and our app is starting in our dev container:
```

Comment:
Try to avoid using "see" as it isn't accessible language.
I think above you can remove "we can see".
"A new instance of VS Code lanches, where our image is being built and our app is starting...."

Reply:
This is an excellent point. Updated the wording to remove "see."